src/country_finder.rs

```diff
@@ -10,18 +10,11 @@ pub struct Country {
     admin_level: Option<u32>,
 }
 
+#[derive(Default)]
```

Comment:
Is it a good practice to have the default trait as fallback even if it's not use ?

Reply:
I don't think this is an especially bad practice, in this case the `country_finder` module is not publicly exposed so there is no regression in removing it. So I removed it just in doubt, it may save us a few dozen of microseconds at compilation :shrug:

Reply:
I had blindly applied this clippy lint: https://rust-lang.github.io/rust-clippy/master/index.html#derivable_impls